designer/server/src/test/scala/pl/touk/nussknacker/ui/component/DefaultComponentServiceSpec.scala

```diff
@@ -499,14 +499,13 @@ class DefaultComponentServiceSpec extends AnyFlatSpec with Matchers with Patient
       (admin, fraudCustomerDataEnricherComponentId, List((CanceledFraudProcessWith2Enrichers, List(DefaultCustomName, SecondCustomName)))),
       (admin, filterComponentId, List(
         (DeployedFraudProcessWith2Filters, List(DefaultFilterName, SecondFilterName)),
-        (FraudProcessWithSubprocess, List(SecondFilterName)),
-        (FraudSubprocess, List(SubprocessFilterName)),
+        (FraudProcessWithSubprocess, List(SecondFilterName, s"<<fragment>> $SubprocessFilterName"))
```

Comment:
Why did you change it?

Reply:
That is because at this moment I show only component usages for scenarios. That's why I removed line 503.